packages/@o3r/localization/schematics/ng-add/helpers/devtools-registration.ts

```diff
@@ -0,0 +1,23 @@
+import { Rule } from '@angular-devkit/schematics';
+import { registerDevtoolsToApplication } from '@o3r/schematics';
+import * as path from 'node:path';
+import type { NgAddSchematicsSchema } from '../schema';
+
+const DEVTOOL_MODULE_NAME = 'LocalizationDevtoolsModule';
+const DEVTOOL_SERVICE_NAME = 'LocalizationDevtoolsService';
+const PACKAGE_NAME: string = require(path.resolve(__dirname, '..', '..', '..', 'package.json')).name;
+
+/**
+ * Register Devtools to the application
+ * @param tree
+ * @param context
+ * @param options
+ * @param options.projectName
+ */
+export const registerDevtools = (options: NgAddSchematicsSchema): Rule =>
```

Comment:
can this be factorized instead of being present in each module?

Reply:
Not sure to understand what you image, is `registerDevtoolsToApplication` not enough factorization?